code/modules/vehicles/cars/car.dm

```diff
@@ -93,3 +96,116 @@
 		M.visible_message("<span class='warning'>[M] is forced into \the [src]!</span>")
 	M.forceMove(src)
 	add_occupant(M, VEHICLE_CONTROL_KIDNAPPED)
+
+
+/obj/vehicle/sealed/car/civ
+	name = "car"
+	desc = "This used to be a pedestrian-focused station, and now all these cars are ruining it!" //ok boomer
+	icon_state = "car"
+	max_integrity = 300
+	armor = list("melee" = 70, "bullet" = 40, "laser" = 40, "energy" = 0, "bomb" = 30, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)
+	enter_delay = 50 //No speed escapes
+	max_occupants = 5
+	movedelay = 1.15
+	material_flags = MATERIAL_AFFECT_STATISTICS | MATERIAL_COLOR | MATERIAL_ADD_PREFIX
+	var/obj/item/card/id/linked_id = null
+	var/can_drive = FALSE
+
+
+/obj/vehicle/sealed/car/civ/Initialize()
+	var/datum/material/M = pick(subtypesof(/datum/material))
```

Comment:
hm yes today I will ram people with my +9 car of the heavens

Reply:
Obama car obama car obama car

Reply:
I mean, it won't actually affect how much damage you deal by ramming someone with your car, but still.

Is there anyway to harvest the mats from your car?

Are non-plasmamen who happen to roll uranium as their car material kind of screwed?

Are cars made of Obamium immune to damage from most melee attacks?

Reply:
no to all but yes to uranium. you can also blow up plasma cars

Reply:
plasma cars == trucks of peace